conda_forge_tick/update_upstream_versions.py

```diff
@@ -75,10 +80,46 @@ def get_version(self, url):
         return str(ver).replace('-', '_')
 
 
-sources = [PyPI(), CRAN(), Github()]
+
+class RawURL:
+    name = 'RawURL'
+    def get_url(self, meta_yaml):
+        pkg = meta_yaml['name']
```

Comment:
As @CJ-Wright mentioned in #83, this should be `pkg = meta_yaml['feedstock_name']`.